src/kata3.numberToReversedDigits.js

```diff
@@ -1,5 +1,10 @@
 const numberToReversedDigits = (number) => {
-
-}
-
+    number = 12345
+    const initialArray = ('' + number).split('');
+    const reversedArray = [];
+    for (let i = initialArray.length - 1; i >= 0; i--) {
```

Comment:
A for loop works just fine here, but you can really cut down on the amount of code if you chain a `.reverse()` onto your `.split('')`.